### ISSUE TITLE
RSS is not getting generated (prolly requests are blocked by Allegro)

### ISSUE THREAD

**Issue by jagub2** [emptycodes/AllegroRSS#4]
Looks like Allegro detects if requests are generated by automated tool than a human. 
I tried to run this project and all I'm getting are 403s (I got it after debugging exceptions yielded by app.py):

`requests.exceptions.HTTPError: 403 Client Error: Forbidden for url ...`

I got this behavior on 2 IPs. Could you please check it?

**Comment by emptycodes** [comment]
Allegro is looking now on User Agent header. AllegroRSS was using default header of popular Python library for making HTTP requests  so... that was to be expected. As hotfix, I set common header.

**Comment by jagub2** [comment]
I also had to set up Referer header. For `User-Agent`, `fake_useragent` sounds like a nice addition. What do you think? I could prepare a pull request.

**Comment by emptycodes** [comment]
We can try it. If you would prepare a PR, then It will be nice :D
I'm going to finish all issues (like a queue) for the weekend.

**Comment by jagub2** [comment]
After further research, plain random UA won't do the job. For other architectures than WOW64/x86_64 instead of page you get 403 with captcha form. I'll create a pull request with just setting UA for now.
